### PR TITLE
Fix CustomView

### PR DESCRIPTION
**Pull request by arkgast**

### CODE REVIEW

File: examples/demos/customView.js [hunk 1]
```diff
@@ -44,18 +51,23 @@ MyWeek.navigate = (date, action) => {
...
-      defaultDate={new Date(2015, 3, 1)}
-      views={{ month: true, week: MyWeek }}
-      test="io"
-    />
-  </div>
-)
-
-export default CustomView
+
+let CustomView = React.createClass({
```
Comment (tobiasandersen) [inline examples/demos/customView.js]: Could you update this to use es6 instead?

Comment (arkgast) [reply examples/demos/customView.js]: yeah, no problem

**Comment by arkgast** [comment]
Hello @tobiasandersen, I think the example was about to set a custom start/end of a week.

As you can see [here][code] the purpose was to get a range but the [date-arithmetic] api right now doesn't have that method.

[code]: https://github.com/intljusticemission/react-big-calendar/pull/658/files#diff-c25d8e64b4c528fe1e1e86385af983ffR20
[date-arithmetic]: https://www.npmjs.com/package/date-arithmetic

**Comment by tobiasandersen** [comment]
Okay cool, I'll just let @jquense have a look at it as well before merging!

**Pull request merged**